build/Build.cs

```diff
@@ -124,7 +124,11 @@ class Build : NukeBuild
 
             File.WriteAllText(octoVersionText, fullSemVer);
             Console.WriteLine($"##[notice]Release version number: {fullSemVer}");
-            Console.WriteLine($"::set-output name=version::{fullSemVer}");
+            var outputFile = Environment.GetEnvironmentVariable("GITHUB_OUTPUT");
```

Comment:
OctoVersion should do this stuff automatically?

Reply:
So we should drop the `set-output` line and have nothing in this build script?

Reply:
Honestly, dunno :)
Might need some investigation into how it works. I'm not much of a GHA expert.

Reply:
this change means that the output is equivalent to what we had previously, without understanding what OctoVersion should or shouldn't be doing

Reply:
👍 I'll leave it with you to decide what to do :)